common-lib/src/cmr/common/rebalancing_collections.clj

```diff
@@ -4,6 +4,8 @@
   (:require
    [cmr.common.services.errors :as errors]))
 
+;; TODO: move to elastic-utils when indexer-app has code moved into that lib
```

Comment:
Can you add the ticket number of when this work will be done? I found this helpful for other comments I've seen as a dev coming from outside this work

Reply:
I actually have one now, so yes